frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistSettings.tsx

```diff
@@ -94,6 +100,66 @@ function SortedBy({
     )
 }
 
+function ConfirmDeleteRecordings({ shortId }: { shortId?: string }): JSX.Element {
+    const { selectedRecordingsIds, isDeleteSelectedRecordingsDialogOpen, deleteConfirmationText } =
+        useValues(sessionRecordingsPlaylistLogic)
+    const { setIsDeleteSelectedRecordingsDialogOpen, setDeleteConfirmationText, handleDeleteSelectedRecordings } =
+        useActions(sessionRecordingsPlaylistLogic)
+
+    const handleClose = (): void => {
+        setIsDeleteSelectedRecordingsDialogOpen(false)
+        setDeleteConfirmationText('')
+    }
+
+    return (
+        <LemonModal
+            isOpen={isDeleteSelectedRecordingsDialogOpen}
+            onClose={handleClose}
+            title="Confirm deletion"
+            maxWidth="500px"
+        >
+            <div className="space-y-4">
+                <h4>
+                    Are you sure you want to delete {selectedRecordingsIds.length} recording
+                    {selectedRecordingsIds.length > 1 ? 's' : ''}?
+                </h4>
+                <div className="space-y-2">
+                    <label className="text-sm">
+                        To confirm, please type <strong>{DELETE_CONFIRMATION_TEXT}</strong> below:
+                    </label>
+                    <LemonInput
+                        value={deleteConfirmationText}
+                        onChange={setDeleteConfirmationText}
+                        placeholder={DELETE_CONFIRMATION_TEXT}
+                        className="w-full"
+                        autoFocus
+                    />
+                </div>
+                <div className="bg-warning-highlight border border-warning rounded p-2 text-sm">
+                    This action cannot be undone. Deleting recordings won't affect your billing since we charge for
+                    ingestion, not storage.
```

Comment:
```suggestion
                    This action cannot be undone. Deleting recordings doesn't affect your billing.
```

maybe less is more...?